externs/shaka/player.js

```diff
@@ -1299,13 +1342,7 @@ shaka.extern.ManifestConfiguration;
  *   disableAudioPrefetch: boolean,
  *   disableTextPrefetch: boolean,
  *   disableVideoPrefetch: boolean,
- *   liveSync: boolean,
- *   liveSyncTargetLatency: number,
- *   liveSyncTargetLatencyTolerance: number,
- *   liveSyncPlaybackRate: number,
- *   liveSyncMinPlaybackRate: number,
- *   liveSyncPanicMode: boolean,
- *   liveSyncPanicThreshold: number,
+ *   liveSync: shaka.extern.LiveSync,
```

Comment:
any concern from changing this from a boolean into an object?

Reply:
No, but you have to fix the deprecate so that it looks to see if it is a boolean, and if it is a boolean, it converts it to the object. Now I will make the change and push it into your PR.

Reply:
ah, totally forgot to make sure this happens. Thanks!